src/NUglify/Html/HtmlWriterToHtml.cs

```diff
@@ -67,7 +67,36 @@ protected override void WriteStartTag(HtmlElement node)
                 lastNewLine = true;
             }
 
-            base.WriteStartTag(node);
```

Comment:
Why the following code couldn't be modified into the `HtmlWriterBase`?

Reply:
In addition to checking element kind and XML namespace level, we should check `RemoveOptionalTags` option from settings to make a decision about writing `" /"`. Unfortunately, I don't see a better way to pass this logic to `HtmlWriterBase`.